File: Basic/Math/all_divisors_of_a_num.cpp
#include <cmath>
#include <iostream>
#include <vector>
using namespace std;
vector<int> getDivisors(int n)
{
    vector<int> ans;
    for (int i = 1; i <= sqrt(n); i++)
    {
        if (n % i == 0)
        {
            ans.push_back(i);
            if (i != n / i)
            {
                ans.push_back(n / i);
            }
        }
    }
    sort(ans.begin(), ans.end());
    return ans;
}

File: Basic/Math/all_divisors_of_a_num.py
import math


def get_divisors(n):
    ans = []
    for i in range(1, int(math.sqrt(n)) + 1):
        if n % i == 0:
            ans.append(i)
            if i != n // i:
                ans.append(n // i)
    ans.sort()
    return ans
